config/routes.rb

```diff
@@ -15,5 +15,7 @@
   get '/general_shopping_lists', to: 'general_shopping_lists#index'
 
   resources :foods
-  resources :recipes
+  resources :recipes do
+    resources :recipe_foods
+  end
```

Comment:
- [ ] I've noticed that this /foods page is visible even when there is no logged-in user. This page should require a user to be logged in and only that user should be able to see foods, recipes...
- You should notice that the button `add-food` shows an error if there is no logged-in user.